### PR TITLE
Status and Shop links in footer

### PR DESCRIPTION
**Pull request by rishsingh07**
Added status and shop link in footer

### CODE REVIEW

File: src/features/projects/components/maps/Credits.tsx [hunk 1]
```diff
@@ -44,6 +44,20 @@ export default function Credits({ setCurrencyCode }: Props): ReactElement {
                     {`🌐 ${language ? language.toUpperCase() : ''
                         } • ${selectedCurrency}`}
                 </div>
+                <a
+                    rel="noopener noreferrer"
+                    href={`https://status.plant-for-the-planet.org/`}
+                    target={'_blank'}
+                >
+                    {t('common:status')}
```
Comment (norbertschuler) [inline src/features/projects/components/maps/Credits.tsx:52]: The status item is also not part of our bigger footer? Should it or shouldn't it?

Big footer:

Bildschirmfoto 2021-04-05 um 12 16 29

Small footer:

Bildschirmfoto 2021-04-05 um 12 16 40

Comment (rishsingh07) [reply src/features/projects/components/maps/Credits.tsx:52]: @harshvitra

Comment (harshvitra) [reply src/features/projects/components/maps/Credits.tsx:52]: Yes we might add it to the bigger footer also

**Comment by sagararyal** [comment]
@rishsingh07 status should always point to status.pp.eco regardless of tenants.

**Comment by sagararyal** [comment]
@rishsingh07 url is status.pp.eco

This will make sure site is still up even if DNS errors affect -planet.org

Please replace these as well
Screen Shot 2021-04-12 at 3 21 41 PM

**Pull request merged**